### PR TITLE
Added ArchOS installation info

### PR DESCRIPTION
**Pull request by KeneePatel**
## Relevant Issues
Relevant to issue #3   

## Description
Just added info for installing in a arch based system.

## Reasoning
Made a PKGBUILD and then uploaded the same to AUR.

## Screenshots
image

## Did you:

- [x] Join the [**Piston CLI Discord Community**](https://discord.gg/c7dZ4zdGQT)?
- [x] If dependencies have been added or updated, run `pipenv lock`?
- [x] **Lint your code** (`pipenv run lint`)?
- [x] Set the PR to **allow edits from contributors**?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -45,6 +45,17 @@ nix-env -i piston-cli
 # using nix profile (in nixUnstable)
 nix profile install nixpkgs#piston-cli
 ```
+#### For Arch/ArchBased
+##### With yay
+```bash
+yay piston-cli
+```
```
Comment (ShotgunFlamez) [inline README.md:52]: ??? This Is Stupid, And Ur A Foot

**Review by Shivansh-007 — APPROVED** [review]
Awesome! Looks Good to Me. I asked few of my friends to test it and it works great!

**Comment by Shivansh-007** [comment]
https://aur.archlinux.org/packages/piston-cli-git/ Can we change the maintainer to @SuperMaZingCoder and me here,  and if possible remove the last contributor?

**Comment by RohanJnr** [comment]
`ModuleNotFoundError: No module named 'prompt_toolkit'` when I tried piston -h after installing using `yay`

**Review by Shivansh-007 — CHANGES_REQUESTED** [review]
Few comments on your PKGBUILD,
```bash
# Maintainer: Kenee Patel  <patelkenee2804 + aur at gmail <.> com>
pkgname=piston-cli-git
_pkgname=piston-cli
pkgver=r71.094d30a
pkgrel=1
pkgdesc="A universal shell supporting 36+ languages, in terminal code editor with syntax highlight, code compiler (36+ languages). It can run code from pastebin links and files too. "
arch=('any')
url="https://github.com/Shivansh-007/piston-cli.git"
license=('MIT')
depends=('python' 'python-pip' 'python-rich')
makedepends=('git')
optdepends=()
provides=('piston-cli')
conflicts=('piston-cli')
source=('git'+$url)
sha256sums=('SKIP')

build() {
	cd "$_pkgname"
	python setup.py build
}

package() {
	cd "$_pkgname"
	python setup.py install --root="${pkgdir}/" --optimize=1 --skip-build
	install -Dm644 LICENSE "${pkgdir}/usr/share/licenses/${pkgname}/LICENSE"
}

pkgver() {
    cd "$srcdir/$_pkgname"
    printf "r%s.%s" $(git rev-list --count HEAD) $(git rev-parse --short HEAD)
}
```

1. Change the maintainer's name and email address to Supermazingcoder's.
2. Change the depends and make depends to:
	```bash
	depends=('python-pygments'
	'python-requests'
	'python-prompt_toolkit'
	'python-rich'
	'python-argparse')
	makedepends=('python-setuptools') 
	```
3. We would be shifting to use github releases so we would need support for that, so you can have a look at https://github.com/archlinux/svntogit-community/blob/packages/httpie/trunk/PKGBUILD (httpie's PKGBUILD). 
4. Add the PKGBUILD to `packaging/` or `scripts/` maybe.

**Comment by KeneePatel** [comment]
Okay I was meaning to change it beforehand just putted mine in as temporary email only.

File: packaging/PKGBUILD [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+arch=('any') # architecture, as this is python based, this can be any.
+url="https://github.com/Shivansh-007/piston-cli.git" # the url to the source on github
+license=('MIT')
+# Depenencies should be mentioned and updated here.
+depends=(	'python-pygments'
+			'python-requests'
+			'python-prompt_toolkit'
+			'python-rich'
+			'python-argparse'
+)
```
Comment (Shivansh-007) [inline packaging/PKGBUILD:16]: Indentation issues ^
```suggestion
depends=(
    "python-pygments"
    "python-requests"
    "python-prompt_toolkit"
    "python-rich"
    "python-argparse"
)

```

**Review by Shivansh-007 — APPROVED** [review]

**Pull request merged**